### PR TITLE
python3Packages.pyhomematic: 0.1.71 -> 0.1.72

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116339` at c34e9c68 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/c1ada469a364c6ba5328755ecfda298c">python38Packages.pyhomematic</a></li></li>
    <li><a href="https://gist.github.com/ffc947dbb1bd90071a995a4712680a8c">python39Packages.pyhomematic</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/pyhomematic/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
   disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0z09bw2mlhg7n9jyqmcyir306wpxr5nw1qsp5ps2iaw1qnyz5s9n";
+    sha256 = "sha256-odRBA7kEGNnIzeRpmhxnHVfRJGm+I6Rek7/ADfKO8Rs=";
   };
 
   # PyPI tarball does not include tests/ directory
```
Comment (mweinelt) [inline pkgs/development/python-modules/pyhomematic/default.nix:14]: The linked issue has been closed. Can we get the tests up and running?

**Review by mweinelt — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pyhomematic/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/pyhomematic/default.nix:14]: We could set the delay to a high value, but who knows what would be high enough for Hydra, aarch64, ...

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116339` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python38Packages.pyhomematic</li>
<li>python39Packages.pyhomematic</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python38Packages.pyhomematic:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
</details>

File: pkgs/development/python-modules/pyhomematic/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/pyhomematic/default.nix:14]: If not please at least add a pythonImportsCheck.

Comment (dotlambda) [reply pkgs/development/python-modules/pyhomematic/default.nix]: I enabled tests without changing the delay. I'll keep an eye out for build failures on Hydra.

**Pull request merged**